### ISSUE TITLE
CatObj is backwards

### ISSUE THREAD

**Issue by malsobrook** [Boa-Net/Tyrus#1]
The CatObj is actually a Tyrus Object.

I believe you would want the reverse. Create a Cat Object that when you create, you assign to a variable named Tyrus. This way you can have more cats, which are Cats, not Tyruses.

**Comment by Boa-Net** [comment]
This is true. I'll change it the class to name Cat, but I don't intend to be calling multiple cats in one run of this code. From what I've done so far, there's only one cat you take care of as the user, so I figured there'd be only one Cat Object. I'm mostly just using it as a way to access methods, and I don't really understand it yet.
